docs/apis-tools/operate-api/overview.md

```diff
@@ -42,6 +42,13 @@ For SaaS: `https://${REGION}.operate.camunda.io/${CLUSTER_ID}/swagger-ui.html`,
 Find your **region Id** and **cluster Id** under **Connection information** in your client credentials (revealed when you click on your client under the **API** tab within your cluster).
 :::
 
+#### Limitation for Operate API v.1
```

Comment:
I think this would be a better fit for a Warning box (see [here](https://docs.camunda.io/docs/self-managed/operational-guides/monitoring/metrics/#prometheus) for an example) than a dedicated subsection.